lib/orbf/rules_engine/fetch_data/contract_orgunits_resolver.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Orbf
+  module RulesEngine
+    class ContractOrgunitsResolver
+      def initialize(package, pyramid, main_orgunit, contract_service, period)
+        @package = package
+        @pyramid = pyramid
+        @main_orgunit = main_orgunit
+        @contract_service = contract_service
+        @main_orgunit_contract = contract_service.for(main_orgunit.ext_id, period)
+        @period = period
+      end
+
+      def call
+        unless @main_orgunit_contract
```

Comment:
This is the 'no contract in place for this period' escape clause right? Should we worry/know about this, if we try to run it and we don't have a contract in place but we do have a contract_service defined on the project?